config/base/kustomization.yaml

```diff
@@ -40,30 +38,4 @@ patchesStrategicMerge:
 # Uncomment 'CAINJECTION' in crd/kustomization.yaml to enable the CA injection in the admission webhooks.
 # 'CERTMANAGER' needs to be enabled to use ca injection
 #- webhookcainjection_patch.yaml
-vars:
-  - name: CERTIFICATE_NAMESPACE # namespace of the certificate CR
-    objref:
-      kind: Certificate
-      group: cert-manager.io
-      version: v1
-      name: serving-cert # this name should match the one in certificate.yaml
-    fieldref:
-      fieldpath: metadata.namespace
-  - name: CERTIFICATE_NAME
-    objref:
-      kind: Certificate
-      group: cert-manager.io
-      version: v1
-      name: serving-cert # this name should match the one in certificate.yaml
-  - name: SERVICE_NAMESPACE # namespace of the service
```

Comment:
If I see correctly SERVICE_NAMESPACE and SERVICE_NAME are used in ../certmanager (and only there? please double check).

Then they have to stay here

Reply:
Hm. I'm not entirely sure to be honest

I think these two env vars get data from the service.yaml and then they are used in certificate.yaml

Reply:
Hm, looks like it works. I'll do a bit more testing

Reply:
All good. Diffed the generated manifests befor/after your PR. Looks perfect now